### PR TITLE
fix(attributes): remove extra line break

### LINKED ISSUE #1884: PHP8 - Multi line attributes - Is the empty line afterwards by design?
Using PHP8 with mutli line attributes result in formatting like this by prettier:

```php
    #[
        ORM\Column(...),
        ORM\CustomIdGenerator(class: ...),
        ORM\GeneratedValue(strategy: 'CUSTOM'),
        ORM\Id
    ]

    private string $id;
```

however, for single line attributes it is like this:

```php
    #[ORM\Column(...)]
    private string $id;
```

**Prettier 2.5.1**

**PHP Plugin 0.17.6**

`.prettierrc`
```json
{
    "$schema": "http://json.schemastore.org/prettierrc",
    "arrowParens": "always",
    "braceStyle": "psr-2",
    "bracketSameLine": false,
    "bracketSpacing": true,
    "phpVersion": "8.0",
    "printWidth": 120,
    "singleQuote": true,
    "tabWidth": 4,
    "trailingComma": "es5",
    "trailingCommaPHP": true
}

```

It is weird that the multi-line syntax adds an empty new line afterwards, but the single line does not. 

Is this by design or a bug?

This issue might be related: https://github.com/prettier/plugin-php/issues/1885

### PR DESCRIPTION
**Pull request by czosel**
Fixes #1884

### CODE REVIEW

File: tests/attributes/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -81,7 +81,7 @@ class D
      */
     #[I, J]
     function k(#[L] int $m): callable {
-        return #[N, O]  #[P] fn(#[Q] int $r) => $r * 2;
+        return #[N, O] #[P] fn(#[Q] int $r) => $r * 2;
```
Comment (czosel) [inline tests/attributes/__snapshots__/jsfmt.spec.js.snap:84]: drive-by fix: remove double space between multiple attr group if `inline==true`

**Comment by Epskampie** [comment]
Thanks for fixing this! Would love to see it merged.

**Comment by czosel** [comment]
@cseufert could you take a quick look? :slightly_smiling_face:

File: src/printer.js [hunk 2]
```diff
@@ -1215,13 +1215,13 @@ function printAttrs(path, options, print, { inline = false } = {}) {
   path.each((agPath) => {
     const attrGroup = ["#["];
     if (allAttrs.length > 0) {
-      allAttrs.push(inline ? " " : hardline);
+      allAttrs.push(inline ? "" : hardline);
     }
```
Comment (cseufert) [inline src/printer.js:1219]: Could this be simplified to 
```js
    if (!inline && allAttrs.length > 0) {
      allAttrs.push(hardline);
    }
```

**Review by cseufert — APPROVED** [review]
Look great, just could clean up conditions for empty non-inline attrgroup

File: src/printer.js (hunk 2 above)
Comment (czosel) [reply src/printer.js]: Absolutely - fixed.
